exercises/sum-of-multiples/.meta/generator/sum_of_multiples_case.rb

```diff
@@ -0,0 +1,8 @@
+require 'generator/exercise_case'
+
+class SumOfMultiplesCase < Generator::ExerciseCase
+  def workload
+    indent_lines(["assert_equal #{expected}, SumOfMultiples.new" \
+                  "(#{factors.join(', ')}).to(#{limit})"] , 4)
```

Comment:
1) I like to try really hard to avoid line continuations like this.
2) Did you know `ExerciseCase` has an `assert_equal` helper method?